src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -986,9 +986,30 @@ const WorkflowView = _.flow(
     const selectedTableName = modifiedConfig.dataReferenceName ? modifiedConfig.rootEntityType : undefined
     const selectionMetadata = selectedTableName ? selectedSnapshotEntityMetadata : entityMetadata
     const attributeNames = _.get([modifiedConfig.rootEntityType, 'attributeNames'], selectionMetadata) || []
+
+    const getSuggestionsForAttributesOfSetMembers = () => {
+      if (!modifiedConfig.rootEntityType.endsWith('_set')) {
+        return []
+      }
+      const memberEntityType = modifiedConfig.rootEntityType.slice(0, modifiedConfig.rootEntityType.length - 4)
```

Comment:
Is this correct? If so, it perhaps makes the intent more clear.
```suggestion
      const memberEntityType = modifiedConfig.rootEntityType.slice(0, modifiedConfig.rootEntityType.length - '_set'.length)
```

Reply:
This was based on Terra's behavior when uploading a membership TSV. I don't know if it's formally documented anywhere, but it looks like there's some magic based on the `_set` suffix. I can upload a TSV with column headers `entity:test_set attribute` and the resulting table gets a `tests` column even if there is not a `test` table in the workspace. Thus, an additional check after this to see if the member entity type derived here actually exists in the workspace.